models/project_board.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/timeutil"
+
+	"xorm.io/xorm"
+)
+
+type (
+	// ProjectBoardType is used to represent a project board type
+	ProjectBoardType uint8
+
+	// ProjectBoardList is a list of all project boards in a repository
+	ProjectBoardList []*ProjectBoard
+)
+
+const (
+	// None is a project board type that has no predefined columns
+	None ProjectBoardType = iota
```

Comment:
ProjectBoardNone is better and the same as below

Reply:
good catch!